pom.xml

```diff
@@ -794,6 +794,11 @@
         <artifactId>jamon-runtime</artifactId>
         <version>${jamon-runtime.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.postgresql</groupId>
+        <artifactId>postgresql</artifactId>
+        <version>42.2-16</version>
```

Comment:
Just for other reviewers, here is the list of valid versions of the `postgresql` artifact in the `org.postgresql` group: https://search.maven.org/artifact/org.postgresql/postgresql

Reply:
Actually @EmilyM1 we may want to avoid hardcoding that particular postgres jdbc version and use a variable like the other pom dependencies that are listed here.

Reply:
Did the Metering Hive fork image build successfully with this change?

Reply:
Did beeline use the update JDBC driver? Hive?